crates/edr_solidity_tests/src/multi_runner.rs

```diff
@@ -189,6 +196,13 @@ impl<
 
         let test_options: TestOptions = TestOptions { fuzz, invariant };
 
+        if generate_gas_report {
```

Comment:
Suggestion: It would be good to document these side-effects in the solidity test runner config's API docs